lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+class Menu
+
+  attr_reader :items
+
+  def initialize
+    @items = {burger: 10, pizza: 8, soup: 7}
+  end
+
+  def view_menu
+    items.each { |k,v| puts "#{k}: #{v} GBP" }
```

Comment:
Space missing after comma.<br>Do not write to stdout. Use Rails' logger if you want to log.